### PR TITLE
Programmatic skip scenario

### PR DESCRIPTION
**Pull request by PiotrNestor**
Add skipScenario indication to ExecutionResult for skipping scenario during runtime

### CODE REVIEW

File: spec.proto [hunk 1]
```diff
@@ -311,6 +311,8 @@ message ProtoExecutionResult {
     string failureScreenshotFile = 11;
     /// Path to the screenshot files captured using Gauge screenshsot API.
     repeated string screenshotFiles = 12;
+    /// Flag to indicate skipping of the scenario execution
+    repeated bool skipScenario = 13;
```
Comment (sriv) [inline spec.proto:315]: Why is this a repeated bool? This will translate to an array of Boolean at the language end.

**Review by sriv — COMMENTED** [review]
Why

File: spec.proto (hunk 1 above)
Comment (PiotrNestor) [reply spec.proto]: Yes, I've corrected this issue ...

**Review by sriv — APPROVED** [review]

**Comment by sschulz92** [comment]
> Is this feature available for python?

See https://github.com/getgauge/gauge-python/issues/390

**Pull request merged**